nuitka/plugins/standard/standard.nuitka-package.config.yml

```diff
@@ -9482,3 +9482,19 @@
         '_skip_under_py2': "'(lambda test_method: None)'"
         '_skip_under_py3k': "'(lambda test_method: None)'"
       when: 'not use_unittest'
+
+- module-name: 'roboflow.core.version'
+  anti-bloat:
+    - description: 'remove IPython reference'
+      no-auto-follow:
+        'IPython': 'ignore'
+      when: 'not use_ipython'
+
+- module-name: 'roboflow.models.object_detection'
+  anti-bloat:
+    - description: 'remove IPython reference'
+      replacements_plain:
+        'import ipywidgets.widgets as widgets': 'raise ImportError'
```

Comment:
I added no-auto-follow for exactly this purpose with the added benefit that it's a global config, i.e. you just attach it to a package. If a package is optional for one submodule, very generally it is for all uses and when things move, we then don't care.

As a result it's more simple:

```
- module-name: 'roboflow'
  anti-bloat:
    - description: 'remove IPython reference'
      no-auto-follow:
        'IPython': 'ignore'
        'ipywidgets': 'ignore'
      when: 'not use_ipython'
```